tools/hash.bat

```diff
@@ -1,11 +1,6 @@
 @echo off
 set DIR=%~dp0
 
-echo "**************************************************************************"
-echo "** This tool will be deprecated in the next major release of OpenSearch **"
-echo "** https://github.com/opensearch-project/security/issues/1755           **"
-echo "**************************************************************************"
-

```

Comment:
Why remove? IMO we still want to deprecate

I think working on this CR has been a good example of why writing two different implementations stinks